app/Libraries/Score/FetchDedupedScores.php

```diff
@@ -31,6 +31,8 @@ public function all(): array
 
         while ($hasNext) {
             if ($nextCursor !== null) {
+                // FIXME: cursor value returned is 0/1 but elasticsearch expects false/true...
```

Comment:
0/1 works but only if it's actual int instead of casted to string. Maybe look into removing the casting in `getSortCursor` especially as `cursor_string` is now the recommended way to pass around cursors and thus isn't affected by javascript limitations